src/map.js

```diff
@@ -99,38 +109,43 @@ export const generateMapHTML = function (total) {
     />
   ));
 
-  return ReactDOMServer.renderToString(
-    <html>
-      <head>
-        <style>{styles.toString()}</style>
-      </head>
-      <body>
-        <h1>
-          <a href="/">
-            <img
-              id="logobanner"
-              src="/images/background_and_menus/logobanner.svg"
-            />
-          </a>
-        </h1>
-        <div id="feedback">
-          <a href="https://github.com/GitTerraGame/GitTerra/issues/new?template=feedback.md&amp;labels=feedback">
-            How can we make this game better?
-          </a>
-        </div>
-        <div
-          style={{
-            position: "absolute",
-            width: `${highestIsoX - lowestIsoX + tileWidth}px`,
-            height: `${highestIsoY + highestTileHeight - tileHeight}px`,
-            left: `50%`,
-            marginRight: `-50%`,
-            transform: `translate(-50%, 0)`,
-          }}
-        >
-          {tileImages}
-        </div>
-      </body>
-    </html>
+  return (
+    "<!doctype html>" +
+    ReactDOMServer.renderToString(
+      <html>
+        <head>
+          <style>{styles.toString()}</style>
+          <title>Your Repo Map | GitTerra</title>
+        </head>
+        <body>
+          <GoogleTag googleId={googleId}></GoogleTag>
```

Comment:
this will not pass `propTypes` check if `googleId` is `null` (e.g. there is no `config.json` with this value set).

you might want to use something like
```suggestion
          {googleId && <GoogleTag googleId={googleId}></GoogleTag>}
```